exercise_4

rominamedina opened the pull request:
method that when entering the age shows a sequence of ascending numbers from 1 to the entered age

rominamedina committed:
exercise_4

method that when entering the age shows a sequence of ascending numbers from 1 to the entered age

barondavid commented on exercise_4.rb:
```diff
@@ -1,2 +1,14 @@
+
+end
+
+show_age
```
add a blank line here

barondavid commented on exercise_4.rb:
```diff
@@ -1,2 +1,14 @@
+
+  age = gets.chomp.to_i
+
+  1.upto(age) { |i| puts i }
```
why you can't use a range?

barondavid commented on exercise_4.rb:
```diff
@@ -1,2 +1,14 @@
\ No newline at end of file
+#(desde 1 hasta su edad).
+
+def show_age
```
do we don't need to get a parameter to do this?

rominamedina committed:
Changes in exercise 4

The use of method upto is changed to a for loop (with range)
A parameter age is added when calling the method show_years_old

rominamedina committed:
exercise 5

Method to invert a number

rominamedina committed:
exercise 7

A method that receives a positive integer and writes all primes numbers up to the number entered
